src/app/components/event-selection.tsx

```diff
@@ -0,0 +1,39 @@
+const eventMap = {
+    bath: { label: "took a bath" },
+    game: { label: "played games" },
+    friend: { label: "met a friend" },
+    housework: { label: "did housework" },
+    illness: { label: "not feeling well" },
+    disput: { label: "argued with someone" },
+    sad: { label: "something sad happened" },
```

Comment:
I'd recommend you mix your current keys in general with a verb to make them more specific.

E.g. `friend` could be used for `made a new friend` or what you currently have with `met a friend`